to_srt.py

```diff
@@ -52,7 +52,7 @@ def append_subs(start, end, prev_content):
         append_subs(prev_time["start"], prev_time["end"], prev_content)
         prev_time = {"start": start, "end": end}
         prev_content = [content]
-    append_subs(start, end, prev_content)
+        append_subs(start, end, prev_content)

```

Comment:
I see the format is not very consistent. I got `begin="241910001t" end="280280000t"`and you `begin="00:00:21.040" end="00:00:25.318"`. They are also adding some HTML tags. There are quite a few more differences, but I think I got it working for your case now. Want to do a pull and try again? :D